templates/topic/index.json-ld

```diff
@@ -17,7 +17,7 @@
 	"modified": "2024-08-27",
 	"componentName": "topic",
 	"status": "stable",
-	"version": "2.1",
+	"version": "3.0",
```

Comment:
Set a minor version for now (v2.2) as we do allow both pattern for now, but demote the version 2.1